### PR TITLE
Add heapdict recipe

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

**Comment by mrocklin** [comment]
I don't know why the recipe isn't linting correctly.

**Comment by mrocklin** [comment]
Ah, I just realized that this is on the default channel.  Closing for now.  My willingness to find what's going on just went way down :)

File: recipes/heapdict/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  run:
+    - python
+
+test:
+  imports:
+    - heapdict
+
+about:
+    home: http://stutzbachenterprises.com/
```
Comment (ocefpaf) [inline recipes/heapdict/meta.yaml]: BTW here is the problem. The indentation is wrong :wink: 

To want me to finish this recipe for you?

File: recipes/heapdict/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - heapdict
+
+about:
+  home: http://stutzbachenterprises.com/
+  license: BSD
```
Comment (ocefpaf) [inline recipes/heapdict/meta.yaml]: It seems to be `BSD 3-Clause`

**Comment by ocefpaf** [comment]
I used the GitHub source in my recipe. Here it is in case you want to copy it here. If not I can sent it later to staged-recipes. (I've been meaning to do it for a while now.)

```yaml
{% set version = "1.0.0" %}

package:
    name: heapdict
    version: {{ version }}

source:
    fn: heapdict-{{ version }}.tar.gz
    url: https://github.com/DanielStutzbach/heapdict/archive/v{{ version }}.tar.gz
    sha256: 4c8e581f1651ca78da35820fc80b0d50fc060032517f2c7efd9169e204db3793

build:
    number: 0
    script: python setup.py install --single-version-externally-managed --record=record.txt

requirements:
    build:
        - python
        - setuptools
    run:
        - python

test:
    imports:
        - heapdict

about:
    home: http://stutzbachenterprises.com/
    license: BSD 3-Clause
    summary: 'A Heap MutableMapping.'

extra:
    recipe-maintainers:
        - ocefpaf
```

**Comment by mrocklin** [comment]
I would be very happy to yield control of this to you :)

**Comment by ocefpaf** [comment]
No problem. Sending mine now.

**Pull request closed without merging**